iphone/Classes/TiUIOptionDialogProxy.m

```diff
@@ -99,17 +101,22 @@ -(void)show:(id)args
             }
             curIndex++;
         }
-        
+
         if ([TiUtils isIPad] && (cancelButtonIndex == -1)) {
             UIAlertAction* theAction = [UIAlertAction actionWithTitle:@"Cancel"
                                                                 style:UIAlertActionStyleCancel
                                                               handler:^(UIAlertAction * action){
                                                                   [self fireClickEventWithAction:action];
                                                               }];
+            if (!IS_NULL_OR_NIL(imagesOptions)) {
+                UIImage *image = [ [UIImage imageNamed:[imagesOptions objectAtIndex:curIndex]] imageWithRenderingMode:UIImageRenderingModeAlwaysOriginal];
+                [theAction setValue:image forKey:@"image"];
+            }
+
```

Comment:
@hansemannn I tried to understand TiHelpers, can you provide me sample / doc to create images in a right way?
Thanks!